src/aleph/vm/orchestrator/views/operator.py

```diff
@@ -87,6 +89,27 @@ async def stream_logs(request: web.Request) -> web.StreamResponse:
             execution.vm.unregister_queue(queue)
 
 
+@cors_allow_all
+@require_jwk_authentication
+async def operate_logs(request: web.Request, authenticated_sender: str) -> web.StreamResponse:
```

Comment:
`operate_logs` seems a weird name :thinking: 

```suggestion
async def get_vm_logs(request: web.Request, authenticated_sender: str) -> web.StreamResponse:
```

Reply:
I don't like it much but it stay consistant with the other naming of the method